gerrit_python_tools/gerrit.py

```diff
@@ -624,13 +682,30 @@ def send_upstream(self, downstream, upstream):
         approvals = self.get_approvals(downstream.SSH())
 
         # Check to see if comment has necessary approvals.
-        if not self.is_upstream_approved(approvals):
-            msg = ("Could not send to upstream: One or more labels"
-                   " not approved.")
+        if self.is_forced():
+            if not self.is_release_approved(approvals):
+                msg = "Could not send to upstream: Release not approved."
+                logger.debug("Change %s: %s" % (self.change_id, msg))
+                ssh.exec_once('gerrit review -m %s %s'
+                              % (pipes.quote(msg), self.revision))
+                return
+            msg = ("***WARNING***\n%s (%s) has requested a forced upstream "
+                   "push. Bypassing all votes except for Release...\n\n"
```

Comment:
I want to notice it even if i'm not specifically looking for it. While I trust our team to do the right thing, we are not the only ones who have access to this system.  ANYONE in NetApp can register and comment on these patches.  Also, this is just me being paranoid, usernames can be changed in the database and that will filter out to existing comments as well.  Putting the name and email in the message was a deliberate decision.  All I'm doing is putting an alarm on the Emergency Exit.